### ISSUE TITLE
Errors not shown

### ISSUE THREAD

**Issue by jayvdb** [QuickenLoans/ramllint#53]
If I run `ramllint filename`, I expect to see any errors.  This occurred in 1.2.3 and earlier.

As of ramllint 1.2.4 and ramllint 1.3, I only see errors if I run `ramllint -l error filename`.

Also related, if I have an raml file which has errors, but I use level info, no errors are shown, and the ascii green message says there are no errors were *found*.

```
$ ./node_modules/.bin/ramllint -l info /tmp/bad.raml`

Looking good; no error(s) found.
```

There were errors found ; they just were not printed to the console.

**Comment by kalisjoshua** [comment]
Sorry for the recent problem. If you could provide a sample of what RAML you are running that would help a lot. I will dive into this today and see what I can figure out on my own.

**Comment by jayvdb** [comment]
Any invalid RAML will do. The problem isnt about specific errors ; it is about the command line options needed to select which errors are emitted.

**Comment by kalisjoshua** [comment]
When I `git checkout 6b001bd` - which is the current `HEAD` of "master" - and run the `node src/cli.js test/samples/failing-root.raml` I get the following output:

```
error no baseUri
  RAML section - root - must include: version [api_version]
HINT:
version: 1

error no baseUri
  RAML section - root - must include: baseUri [base_uri]
```

... then when I `git checkout 58092e4` - which was before some recent changes - and run `node src/cli.js test/samples/failing-root.raml` I get the same output as above.

What am I missing? If there are issues I want to write some tests to capture them. Any help you can provide would be appreciated.

**Comment by jayvdb** [comment]
`failing-root.raml` is the only file in `test/samples/` which still emits errors.  Try any of the other `failing-*.raml` files or `parse-error.raml`.

```
$ node src/cli.js test/samples/failing-resource.raml
$ node src/cli.js -l error test/samples/failing-resource.raml
error http://example.com/RESOURCE
  RAML section - resource - must include: description [resource_desc]

error http://example.com/RESOURCE
  RAML section (resource) relativeUri violates: only lowercase letters and underscores allowed [url_lower]
HINT:
/income_tax_documents (good)
/incomeTaxDocuments (bad)
/Income-Tax-Documents (bad)
```

**Comment by kalisjoshua** [comment]
Oh. Thank you. I'll work on that.

If you are interested, you could try out the next version; git branch "version2" or git tag "version2-beta".

**Comment by kalisjoshua** [comment]
I think I found the issue and resolved it in #57. I will publish a new npm version shortly.

**Comment by jayvdb** [comment]
Fixed as best I can see.  Thanks.

**Comment by kalisjoshua** [comment]
You're welcome. I'm glad I could fix it.
